### ISSUE TITLE
How to have two different popup in two different pages

### ISSUE THREAD

**Issue by kodjoe** [tylerpearson/CoverPop#22]
Hello i used your plugin in a page and it works very well thank you. 
How can I add another popup in another page  please ?

I've to duplicate different files or i have to modify a part of them ?
thank you

**Comment by tylerpearson** [comment]
I'm not sure I fully understand the question -- easiest way would be to setup the second page the same as the first and not include the same HTML and JS init function.

**Comment by kodjoe** [comment]
I think you understand, I would like to have two different popups in two different pages on my website.
the first popup is for newsletter and the second one for a link to subscription.

So you think the best way to do that is to duplicate files and call them in the second page


Thank you
Mail: crocodeal06@gmail.com

**Comment by tylerpearson** [comment]
Hmm duplicate what files? If they are two different popups, you would need to have two different setups for the HTML so you can do that. You'd be able to reuse the same JS init function if you keep the same class names.

**Comment by kodjoe** [comment]
You totally right that my actual problem. So I need to change the cover ID right ?
But if i just change the coverID in js file, it does't work too ;(

**Comment by tylerpearson** [comment]
no, you wouldn't need to change the id since you have two popups on separate pages

**Comment by kodjoe** [comment]
ah ok so i can have two popup in different pages with the same files (j s ...) that i duplicated in other folder for example

**Comment by kodjoe** [comment]
when I duplicate all files of coverpopup in other folder for example, to call them in an other page. It works yes, the two popup appear when we navigate on the website. 

but there is one problem, when I close one of the popup, the second one doesn't appear too ?!?

**Comment by tylerpearson** [comment]
Hard to follow what's going on, but it sounds like it's likely due to using the same cookie. [You can change the cookie name in settings](https://github.com/tylerpearson/CoverPop/blob/master/CoverPop.js#L31-L32) so that they use different ones.

**Comment by kodjoe** [comment]
yes cookies is the problem, so if i change cookies name on the js file,i need to change something on html file ?

thank you ;D
